inst/include/Rcpp/String.h

```diff
@@ -3,6 +3,7 @@
 // String.h: Rcpp R/C++ interface class library -- single string
 //
 // Copyright (C) 2012 - 2018  Dirk Eddelbuettel and Romain Francois
+// Copyright (C) 2021  Iñaki Ucar
```

Comment:
I would probably have made that 

// Copyright (C) 2012 - 2020  Dirk Eddelbuettel and Romain Francois
// Copyright (C) 2021             Dirk Eddelbuettel and Romain Francois and Iñaki Ucar

which is for example what we have done here (from a quick grep)

Environment.h:// Copyright (C) 2009 - 2013  Dirk Eddelbuettel and Romain Francois
Environment.h:// Copyright (C) 2014 - 2020  Dirk Eddelbuettel, Romain Francois and Kevin Ushey
[...]
exceptions_impl.h:// Copyright (C) 2012 - 2019  Dirk Eddelbuettel and Romain Francois
exceptions_impl.h:// Copyright (C) 2020         Dirk Eddelbuettel, Romain Francois, and Joshua N. Pritikin

Thoughts?